doc/script_commands.txt

```diff
@@ -3507,6 +3507,35 @@ guild specified by the ID. 0 if the character is not a guild master of any guild
 
 ---------------------------------------
 
+*getguildinfo(<guild ID>, <type>);
+
+This function will look up and return the request type of guild information.
+The function will return -1 on failure.
```

Comment:
The command can return a string (for example with GUILDINFO_NAME) or int (GUILDINFO_LEVEL), 
I suggest that the command stops the script on failure since there is no data type defined

```suggestion
```